### PR TITLE
add contribute page

### PR DESCRIPTION
**Pull request by nabdelgadir**
Connect to https://github.com/strongloop/v4.loopback.io/issues/27.

Add contribute page based on [blueprint](https://github.com/strongloop/v4.loopback.io/blob/master/blueprint/lb4-spec-page-contribute.png).

### CODE REVIEW

File: contribute.html [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+                <div class="col-lg-12 text-center">
+                    <h3 class="section-heading" style="padding-top: 48px;">Contribute to LoopBack</h3>
+                    <p>How do you wish to contribute?</p>
+                    <p style="height:24px"></p>
+                </div>
+            </div>
+    
+            <div class="row text-center" style="justify-content: center">
+                <div class="col-sm-12 col-md-6">
+                    <a class="card-2" href="https://github.com/strongloop/loopback-next/issues">
```
Comment (dhmlau) [inline contribute.html]: I think it should point to https://loopback.io/doc/en/contrib/Reporting-issues.html instead.

File: contribute.html [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+        <div class="container">
+            <h3 class="section-heading text-center" style="padding-top: 48px;">Governance</h3>
+            <p class="regular text-center">The LoopBack project governance model follows the sprint and tradition of open source.</p>
+            <p class="large" style="padding-left:190px;padding-right:190px;padding-top:32px;padding-bottom:24px;">
+                LoopBack is an open, inclusive, and tolerant community of people working together to build 
+                a world-class Node framework and tools. We value diversity of individuals and opinions, and seek to operate on
+                consensus whenever possible. We strive to maintain a welcoming, inclusive, and harassment-free environment, 
+                regardless of the form of communication. When consensus is not achievable, we defer to the owners of each 
+                individual module; the powers of the individual owner are kept in check by the ability of the community to 
+                fork and replace dependencies on the individual module and maintainer.</p>
```
Comment (dhmlau) [inline contribute.html]: i'm thinking whether we include https://loopback.io/doc/en/contrib/Governance.html as "see more details". 
@jannyHou @nabdelgadir , what do you think?

File: contribute.html [hunk 3]
```diff
@@ -0,0 +1,271 @@
...
+    <div class="action">
+            <div class="container">
+                <div class="row">
+                    <div class="col-md-7">
+                        <h3 style="color:white">Do you have a blog post or other content on LoopBack?</h3>
+                            <p class="p regular" style="color:white">Send us a link and we'll promote your work.</p>
+                    </div>
+                    <div class="col-md-4">
+                        <a class="btn btn-primary" role="button" style="padding-left:60px; padding-right:60px" 
+                        href="https://github.com/strongloop/strongloop.com">Submit Your Content</a>
```
Comment (dhmlau) [inline contribute.html]: this is a private repo. 
@DaveYVR , how does it work today?  If someone has a LB blog post and wants you to promote, what's the proper way to do it?

File: contribute.html (hunk 2 above)
Comment (nabdelgadir) [reply contribute.html]: screen shot 2018-12-13 at 8 56 00 am
Something like this?

File: contribute.html (hunk 3 above)
Comment (dhmlau) [reply contribute.html]: talked to Dave. Before we figured out what we're going to do for this, let's remove it.  We can modify the page once we find a good and sustainable way. Thanks.

**Pull request merged**